siliconcompiler/core.py

```diff
@@ -3960,47 +3960,11 @@ def _check_display(self):
             self.logger.warning("Setting ['option', 'nodisplay'] to True")
             self.set('option', 'nodisplay', True)
 
-    ###########################################################################
-    def run(self):
+    def _increment_jobs(self):
```

Comment:
increment_job_name

Reply:
Oh ok on a first glance it wasn't obvious to me that that was the purpose.